its/plugin/tests/pom.xml

```diff
@@ -19,9 +19,21 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.sonarsource.orchestrator</groupId>
+      <artifactId>sonar-orchestrator-junit5</artifactId>
+      <version>4.0.0.404</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.junit.jupiter</groupId>
+          <artifactId>junit-jupiter-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
```

Comment:
Please add a comment with "TODO SONARPHP-1423 Update Sonar-Orchestrator to 4.0.0.404"

Reply:
The todo is still missing :-(